website/static/schemas/atmos/1.0/cli.json

```
@@ -0,0 +1,727 @@
+            "items": {
+              "type": "string"
+            }
+          },
+          "excluded_paths": {
+            "type": "array",
+            "description": "Glob patterns for excluded paths",
+            "items": {
+              "type": "string"
+            }
+          },
+          "name_template": {
```

Biome (2.1.2)

[error] The key name_template was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)